### DISCUSSION TITLE
Application's "Take a photo" only is not accessible or usable for online students. Please help!

### DISCUSSION THREAD

**Discussion by katannshaw** [community/community#137949] [Accessibility] [answered]
### Hi everyone,

I would love to join the Global Campus as a GitHub Education Developer, however the application's "Take a photo" tool has completely blocked my ability to do so.

This is because I cannot complete my application for the GitHub Education Developer Pack due to the "Take a photo" tool not being accessible or usable for me and many other users I've noticed posting in the main "github-education" room.

Here's are the issues I'm experiencing:

- I've been taking online classes while working full-time for three semesters now at the accredited [Highland Community College in Kansas](https://highlandcc.edu/), and have been unable to successfully apply for the GitHub Education Developer Pack
- Online students like me don't have student id's to take a photo of, and you rejected my application after I took a photo of me holding up a printout of my community college's official documentation.
- Trying to take a photo with my iPhone of my screen has been impossible and is frustratingly unnecessary when you could simply allow uploads of documents for proof.
- This huge limitation makes this process completely inaccessible for a certain segment of students, and even moreso for those with various disabilities.
- Users who are blind, visually impaired, missing limbs, or with motor disabilities likely cannot easily take a photo of themselves using this tool.
- In the end, all of this ends up blocking my access to apply, and likely others as well.

As a fan of GitHub, I've been very disappointed in this process, including the lack of attention to others who have voiced their frustration without a change in the tool itself. GitHub is a great community and a great product, and I really hope that this issue gets resolved very soon. Thank you for your time and I hope to hear from you soon.

Kat Shaw

**Comment by cehfisher** [comment]
Hello @katannshaw - thank you for taking the time to provide this feedback! Our Accessibility team is actively working on this issue. We will follow-up here with any additional questions or updates. Thanks again!

**Reply by katannshaw** [reply]
Hi @cehfisher Thank you so much! I really appreciate that info.

Kat

**Reply by cehfisher** [reply]
Hello @katannshaw - thank you for working with us directly to help you resolve your issue!

**Comment by katannshaw** [comment, marked as answer]
I'm marking this issue as resolved as the GitHub support staff and Accessibility Team helped me to resolve this issue. I'm now a proud member of the Global Campus as a GitHub Education Developer. Thank you for your help!

**Reply by swilson15** [reply]
Thank you so much, Kat! I'm thrilled we were able to resolve your issue!
